### ISSUE TITLE
Improvements to user details tab

### ISSUE THREAD

**Issue by planemad** [mapbox/osmcha-frontend#157]
This is the most important view to understand the history of the user and designing this well can really make it easier to evaluate the edits of especially experienced users.

<img width="577" alt="screen shot 2017-07-11 at 12 20 37 pm" src="https://user-images.githubusercontent.com/126868/28054703-a31a0128-6633-11e7-8613-35a9c5de44ee.png">

Some quick suggestions:
- Link good and bad changesets to osmcha list
- Would it be possible to show the hdyc mapper type eg. "Legendary Mapper"?

cc @rasagy

**Comment by rasagy** [comment]
>Link good and bad changesets to osmcha list

@planemad this seems like a good idea. Do you think it’s more common to only look at either good/bad changesets or is it more useful to see all changesets by the user (by clicking on `OSMCha`)? cc @manoharuss

>Would it be possible to show the hdyc mapper type eg. "Legendary Mapper"?

@planemad I honestly feel being more objective (by showing the no. of edits) is better than relying on a subjective title that might not have a standard meaning. Are HDYC titles like Legendary Mapper used across OSM tools, and universally understood? For example, is the difference between Heavy vs Great vs Legendary mapper commonly understood?

**Comment by willemarcel** [comment]
> I honestly feel being more objective (by showing the no. of edits) is better than relying on a subjective title that might not have a standard meaning. 

I agree with @rasagy. I think the number of edits is enough, furthermore it would require us to make more one http request and the HDYC needs authentication to get user information now.

**Comment by zlavergne** [comment]
I would love to see more detailed and accurate user stats in this view. For instance, having detailed created, modified, deleted counts for nodes, ways, and relations and accurate number of edits (rather than changesets - your current "edit" count is for changesets rather than total edits).
